LayoutTests/resources/js-test.js

```diff
@@ -70,7 +70,7 @@ var unexpectedErrorMessage; // set by onerror when expectingError is not true
             refNode = getOrCreate("console", "div");
         else
             refNode = parent.firstChild;
-
+        element.style.overflowAnchor = "none";
```

Comment:
Can't you just style all children of the "console" div in anchoring tests? This line is changing behavior for thousands of tests.

Reply:
Removed.